src/utils/image.py

```diff
@@ -248,10 +248,10 @@ def split_patch_contour_on_corners(patch_corners, source_contour):
             min_distance, nearest_edge_type = min(edge_distances)
             distance_warning = "*" if min_distance > 10 else ""
             logger.info(
-                f"boundary_point={boundary_point}\t nearest_edge_type={nearest_edge_type}\t min_distance={min_distance:.2f}{distance_warning}"
```

Comment:
I still see this diff here, can you check why it's coming? Probably need to pull dev branch again